### PR TITLE
Update reset-passwords.md

### PR DESCRIPTION
**Pull request by sergiopadure**
Updated documentation to Microsoft Graph Powershell due to the retirement of the MSOnline/AzureAD modules

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/admin/add-users/reset-passwords.md [hunk 1]
```diff
@@ -130,32 +130,39 @@ These steps work for a business with tens of users. If you have hundreds or thou
...
-    Set-MsolUser –UserPrincipalName –StrongPasswordRequired  $false
+    Get-MgUser -UserId user@contoso.com | ForEach-Object -Process { Update-MgUser -UserId $_.Id -PasswordPolicies DisableStrongPassword }
     ```
+4. You can turn strong password requirements back on by using this command:
+
+    ```powershell
+    Get-MgUser -UserId user@contoso.com | ForEach-Object -Process { Update-MgUser -UserId $_.Id -PasswordPolicies None }
+    ```
+
+Additional information concerning this command can be found here: [https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.users/update-mguser](/powershell/module/microsoft.graph.users/update-mguser)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/reset-passwords.md]:
```suggestion
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/add-users/reset-passwords.md]: This link is broken.

File: microsoft-365/admin/add-users/reset-passwords.md [hunk 2]
```diff
@@ -130,32 +130,39 @@ These steps work for a business with tens of users. If you have hundreds or thou
...
 For overview information, see [Manage Microsoft 365 with PowerShell](../../enterprise/manage-microsoft-365-with-microsoft-365-powershell.md).
-  
-## Force a password change for all users in your business
-
-Check out this great blog post by Vasil Michev, Microsoft MVP: [Force password change for all users in Office 365](https://go.microsoft.com/fwlink/?linkid=853693).
 
 ## Set strong passwords
 
-1. [Connect to Microsoft 365 with PowerShell](/office365/enterprise/powershell/connect-to-office-365-powershell#connect-with-the-microsoft-azure-active-directory-module-for-windows-powershell).
+1. Connect to [Microsoft Graph Powershell](/powershell/microsoftgraph/installation):
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/reset-passwords.md]:
```suggestion
1. Connect to [Microsoft Graph PowerShell](/powershell/microsoftgraph/installation):
```

File: microsoft-365/admin/add-users/reset-passwords.md [hunk 3]
```diff
@@ -130,32 +130,39 @@ These steps work for a business with tens of users. If you have hundreds or thou
 
 5. Enter an email address where you can receive the temporary passwords. You'll need to notify your users what their temporary passwords are.
   
-## Reset business passwords in bulk
+## Password reset at scale
+
+For complete information on how to change passwords at scale or enforce password change at next login via Microsoft Graph Powershell check out this blog post by Padure Sergio: [Force password change for all users in Microsoft 365 - Microsoft Graph Powershell Edition](https://blog.raindrops.dev/blog/force-password-change-for-all-users-in-office-365/)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/reset-passwords.md:135]: We do not reference third-party sites and blogs in Microsoft documentation, therefore this reference should not be included.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/add-users/reset-passwords.md (hunk 3 above)
Comment (sergiopadure) [reply microsoft-365/admin/add-users/reset-passwords.md:135]: Uh, is that a new policy? A lot of docs have links to blogs for a more extensive explanation of a certain subject or additional code

Comment (dstrome) [reply microsoft-365/admin/add-users/reset-passwords.md:135]: There are limited circumstances where Microsoft collaborates with external contributors to author content or include links to external blogs in articles on learn.microsoft.com. Unfortunately, our team doesn't have the resources to review blog posts that are submitted outside of those circumstances.

**Comment by tiburd** [comment]
Closing per dstrome info:
image

cc: @yogkumgit

**Pull request closed without merging**